dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/DefaultMetadataExportService.java

```diff
@@ -599,7 +599,7 @@ public RootNode getMetadataWithDependenciesAsNode( IdentifiableObject object )
         if ( programStageSection == null ) return metadata;
         metadata.putValue( ProgramStageSection.class, programStageSection );
 
-        programStageSection.getProgramStageDataElements().forEach( programStageDataElement -> handleProgramStageDataElement( metadata, programStageDataElement ) );
+        //programStageSection.getProgramStageDataElements().forEach( programStageDataElement -> handleProgramStageDataElement( metadata, programStageDataElement ) ); //TODO can this be left out?
```

Comment:
Yes, PSDE are expanded by default, so they don't need to be included explicitly.